Misc/NEWS.d/next/Windows/2020-01-24-09-15-41.bpo-8901.hVnhGO.rst

```diff
@@ -1 +1,3 @@
 Ignore the Windows registry when the ``-E`` option is used.
+Add Mingw-w64 64-bit windows machine, fix issue
+in https://bugs.python.org/issue40167
```

Comment:
You'll need to create a new NEWS entry file rather than adding this to an existing one. Use [blurb-it](https://github.com/python/blurb_it) or [blurb](https://pypi.org/project/blurb/).